src/main/kotlin/compiler/parser/Parser.kt

```diff
@@ -441,7 +441,7 @@ class Parser(inputTokens: List<Token>) {
 
     // Parse: <expr> +|- <expr>
     private fun pAdd(): N_EXPR? {
-        val next = this::pPower
+        val next = this::pMultiply
```

Comment:
This (all this Parser delta) is me fixing an actual bug I found while writing tests -- power-of operator's precedence was wrong.